ebl-issuance/src/main/resources/standards/eblissuance/messages/eblissuance-v3.0.0-request.json

```diff
@@ -87,70 +87,86 @@
       "UNLocationCode": "NLAMS"
     },
     "invoicePayableAt": {
-      "UNLocationCode": "DKAAR"
+      "UNLocationCode": "NLAMS"
     },
     "partyContactDetails": [
       {
-        "name": "DCSA test person",
-        "email": "no-reply@dcsa.example.org"
+        "name": "Henrik",
+        "phone": "+45 51801234"
+      },
+      {
+        "name": "Henrik",
+        "phone": "+45 51801234"
       }
     ],
     "documentParties": {
       "shipper": {
-        "partyName": "DCSA CTK",
+        "partyName": "IKEA Denmark",
         "partyContactDetails": [
           {
-            "name": "DCSA test person",
-            "email": "no-reply@dcsa.example.org"
+            "name": "Henrik",
+            "phone": "+45 51801234"
+          },
+          {
+            "name": "Henrik",
+            "phone": "+45 51801234"
           }
         ]
       },
       "consignee": {
         "partyName": "CONSIGNEE_LEGAL_NAME_PLACEHOLDER",
         "identifyingCodes": [
           {
-            "codeListProvider": "EPUI",
+            "codeListProvider": "W3C",
             "partyCode": "CONSIGNEE_PARTY_CODE_PLACEHOLDER",
             "codeListName": "CONSIGNEE_CODE_LIST_NAME_PLACEHOLDER"
           }
```

Comment:
I think these need to be turned into a placeholder objects. It is exceedingly likely that the parties do **not** have a fixed `codeListProvider` for this party.

I feel this basically applies to all parties with placeholders in them.